scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -7,11 +7,42 @@
       padding: 20px;
     }
 
+    input[type=checkbox] {
```

Comment:
why do we have styling for CiviCase at all in shoreditch? Shouldn't all of this go into the extension css?

Reply:
@AkA84 This are legacy pages of Civicase which does not have any styles loaded from civicase extension. Even templates of the extension are being called from civicrm-core instead of civicase exntention,(refer: modules/civicrm/ang/crmCaseType). Though, I will flag this to move under extension if possible to @colemanw